editor/inspector/assets/material.js

```diff
@@ -40,9 +44,19 @@ exports.$ = {
     materialDump: '.material-dump',
     useInstancing: '.useInstancing',
     useBatching: '.useBatching',
+    customSection: '.customSection',
+    customPanel: '.customPanel',
 };
 
 exports.methods = {
+    async getCustomInspector() {
```

Comment:
advise use 'hasCustomInspector' as function name

Reply:
Its return value is now used